ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -29,6 +29,7 @@
 /* INCLUDES ******************************************************************/
 
 #include <ntoskrnl.h>
+
```

Comment:
@SergeGautherie Could you create a separate PR just for the fixes in kdbg? Because I have local changes in that file and would like to manually handle that.

Reply:
Done.